.github/workflows/ci.yml

```diff
@@ -423,5 +423,15 @@ jobs:
 
       - name: Test WebApp codebase
         if: steps.web-change.outputs.run == 'true'
-        run: npm run test
+        run: |
+          set -ex
+          # Without `VUE_CLI_TEST` set, the command hangs forever because
+          # it fails to stop the server :'(
+          # (see https://github.com/vuejs/vue-cli/issues/3401)
+          VUE_CLI_TEST=1 npm run test:e2e:headless
+          # Need to run e2e first, as it will trigger compilation of the
+          # libparsec web plugin that is also needed for the unit tests (but
+          # which doesn't trigger it compilation...)
+          npm run test:unit
         working-directory: oxidation/client
+        timeout-minutes: 10
```

Comment:
Why not splitting the this step in 2 ? one for `e2e` and one for `unit-test` ?

Reply:
done.